### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by gaben3722**
# Pull Request Template

## Description

This fixes the readme issue and get the correct version in place.

Fixes # (Readme)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by iknight7000 — APPROVED** [review]
I see the grammatical fixes throughout the read me.

File: README.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ This project has been worked on by many Lambda labs teams over the past 10 month
...
+How does LIME work? For our problem we will utilize the LIME TextExplainer. The TextExplainer generates a lot of texts similar to the document(by removing some words), then trains a white-box classifier that predicts the output of the black-box classifier. This process can be broken down into three simple steps. First, generate text. Second, predict probabilities for these generated texts. Third, train another classifier to predict the output of the black box classifier. While black boxes are hard to approximate, this algorithm works by approximating it in a small neighborhood near the given text in a white-box classifier. Finally, let's look at some visualizations! Below LimeTextExplainer is showing us the weights for each word in an incident report.
```
Comment (ChristopherKchilton) [inline README.md:66]: The readme update is a good addition. Though this update just corrects minor grammar and spelling errors.

**Review by ChristopherKchilton — APPROVED** [review]
I approve the readme update. It is primarily grammar and spelling fixes.

**Review by ahmedseragcodes — APPROVED** [review]
Sensible updates to read-me for better understanding, approved

**Pull request merged**